docs/components/movement-sensor/_index.md

```diff
@@ -418,7 +418,7 @@ orientation = await my_movement_sensor.get_orientation()
 
 **Returns:**
 
-- [(spatialmath.Orientation)](https://pkg.go.dev/go.viam.com/rdk/spatialmath#Orientation): Orientation is an interface used to express the different parameterizations of the orientation of the movement sensor in 3D Euclidean space.

```

Comment:
Here https://pkg.go.dev/go.viam.com/rdk/components/movementsensor#MovementSensor it shows it this way. Why not include this?

Reply:
Good point. I'll add the library in as it can't hurt.

Reply:
Might want to add import in as well?

Reply:
Maybe! I'm curious what everyone thinks about how to add imports.